gh-pages-stub/public/css/styles.css

```diff
@@ -1018,3 +1005,41 @@ table p {
 .small-width {
   max-width: 300px;
 }
+
+/* Form */
```

Comment:
can u, please, remove the opacity for the disabled button & decrease the paddings above/beneath the form

Reply:
@dr3amf4ll

Reply:
& make text color white, grey text is difficult to distinguish on the orange bg.

Reply:
Done